### PR TITLE
[EDU-2771] docs: improve old functions section in Rules Engine

### PR DESCRIPTION
**Pull request by bru-andrade**
### Changes

- Improved explanation of variables as arguments
- Clarified "function" variables
- Fixed Digital Certificates heading hierarchy and section title

### Additional links

[EDU-3000]: https://aziontech.atlassian.net/browse/EDU-3000?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[EDU-2771]: https://aziontech.atlassian.net/browse/EDU-2771?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/main-menu/reference/build/edge-application/rules-engine.mdx [hunk 1]
```diff
@@ -141,6 +143,24 @@ Most mTLS services rely on receiving the user certificate. By using [Real-Time M
 
 <Button href="/en/documentation/products/edge-application/domains/mtls/" text="Learn more about support for mTLS"></Button>
 
+## Variables as arguments
+
+For behaviors that require an argument, you can use the same variables available for the processing phase in which they are available. This way, you can, for example, compose cookies or HTTP headers using data collected durin the request phase, such as the user's device group or their geolocation.
```
Comment (hannah-sk) [inline src/content/docs/en/pages/main-menu/reference/build/edge-application/rules-engine.mdx:148]:
```suggestion
For behaviors that require an argument, you can use the same variables available for the processing phase in which they're available. This way, you can, for example, compose cookies or HTTP headers using data collected during the request phase, such as the user's device group or their geolocation.
```

File: src/content/docs/en/pages/main-menu/reference/build/edge-application/rules-engine.mdx [hunk 2]
```diff
@@ -141,6 +143,24 @@ Most mTLS services rely on receiving the user certificate. By using [Real-Time M
...
+> If `${host}` *is equal* `myhost.com`
+> Then **Add Response Cookie** `cookie-host-value=${host}`
+
+When this rule is active, if the request is made from the host determined in the criteria, the response will return the following header: `Set-Cookie: cookie-host-value=myhost.com`.
+
+Azion also provides special variables that act as functions and take arguments. The following variables can be used to compose **Behavior** arguments:
+
+| Variable | Description |
+| --- | --- |
+| `${cookie_time_offset(number)}` | Returns the current date plus an offset in seconds, entered as an argument, to be used to define the expiration time of a cookie. For example, if you want the cookie to expire in 1 hour from the time it is created, use the *Add Response Cookie* behavior with the argument: `cookie-name=cookie-value; Expires=${cookie_time_offset(3600)}` |
```
Comment (hannah-sk) [inline src/content/docs/en/pages/main-menu/reference/build/edge-application/rules-engine.mdx:161]:
```suggestion
| `${cookie_time_offset(number)}` | Returns the current date plus an offset in seconds, entered as an argument, to be used to define the expiration time of a cookie. For example, if you want the cookie to expire in 1 hour from the time it's created, use the *Add Response Cookie* behavior with the argument: `cookie-name=cookie-value; Expires=${cookie_time_offset(3600)}` |
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/digital-certificates.mdx [hunk 3]
```diff
@@ -33,11 +33,11 @@ Se desejar, você pode utilizar o domínio atribuído para entregar seu conteúd
 
 ---
 
-## Custom certificate
+## Edge certificate
 
 Se você quiser alterar seu domínio, você pode registrar seu próprio certificado TLS (X.509) no [Real-Time Manager](https://manager.azion.com/) sem custo adicional. Para utilização desse recurso, você precisará comprovar a propriedade do domínio.
 
-Para adicionar seu Custom Certificate, você precisará do par Certificado (X.509) e da respectiva chave privada, ambos no formato [ASCII PEM](https://www.google.com.br/search?q=Como+converter+um+certificado+PFX+para+PEM&cad=h). A chave privada não pode estar protegida por *passphrase*.
+Para adicionar seu certificado próprio, você precisará do par Certificado (X.509) e da respectiva chave privada, ambos no formato [ASCII PEM](https://www.google.com.br/search?q=Como+converter+um+certificado+PFX+para+PEM&cad=h). A chave privada não pode estar protegida por *passphrase*.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/digital-certificates.mdx:40]: é uma info diferente da que ta em EN

**Review by hannah-sk — COMMENTED** [review]
adicionei contrações e um comentário

File: src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/digital-certificates.mdx (hunk 3 above)
Comment (bru-andrade) [reply src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/digital-certificates.mdx:40]: Verdade! Vou corrigir isso 😨

File: src/content/docs/en/pages/main-menu/reference/build/edge-application/digital-certificates.mdx [hunk 4]
```diff
@@ -33,11 +33,11 @@ If you wish, you may use the assigned domain to deliver your static content over
 
 ---
 
-## Custom certificate
+## Edge certificate
 
 If you want to use a custom domain, you can register your own TLS certificate (X.509) at no additional cost in [Real-Time Manager](https://manager.azion.com). To use this feature, you'll need to prove that the domain is really yours.
```
Comment (bru-andrade) [inline src/content/docs/en/pages/main-menu/reference/build/edge-application/digital-certificates.mdx:38]:
```suggestion
If you want to use a custom domain, you can register your own TLS certificate (X.509) at no additional cost in [Real-Time Manager](https://manager.azion.com). Register your certificate and private key encoded in [ASCII PEM](https://www.ssl.com/guide/pem-der-crt-and-cer-x-509-encodings-and-conversions/) format. The private key cannot be protected by passphrase.
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/digital-certificates.mdx [hunk 5]
```diff
@@ -33,11 +33,11 @@ Se desejar, você pode utilizar o domínio atribuído para entregar seu conteúd
 
 ---
 
-## Custom certificate
+## Edge certificate
 
 Se você quiser alterar seu domínio, você pode registrar seu próprio certificado TLS (X.509) no [Real-Time Manager](https://manager.azion.com/) sem custo adicional. Para utilização desse recurso, você precisará comprovar a propriedade do domínio.
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/digital-certificates.mdx:38]:
```suggestion
Se você quiser usar um domínio próprio, você pode registrar seu próprio certificado TLS (X.509) e chave privada em formato [ASCII PEM](https://www.ssl.com/guide/pem-der-crt-and-cer-x-509-encodings-and-conversions/) no [Real-Time Manager](https://manager.azion.com/) sem custo adicional. A chave privada não pode estar protegida por *passphrase*.
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/digital-certificates.mdx (hunk 3 above)
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/digital-certificates.mdx:40]:
```suggestion
Quando seu certificado é registrado com a Azion, edge applications com o certificado usam a extensão Server Name Indication (SNI) do protocolo TLS. Verifique a [lista da navegadores com suporte a SNI](https://caniuse.com/#feat=sni).
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/digital-certificates.mdx [hunk 6]
```diff
@@ -33,11 +33,11 @@ Se desejar, você pode utilizar o domínio atribuído para entregar seu conteúd
 
 ---
 
-## Custom certificate
+## Edge certificate
 
-Se você quiser alterar seu domínio, você pode registrar seu próprio certificado TLS (X.509) no [Real-Time Manager](https://manager.azion.com/) sem custo adicional. Para utilização desse recurso, você precisará comprovar a propriedade do domínio.
+Se você quiser usar um domínio próprio, você pode registrar seu próprio certificado TLS (X.509) e chave privada em formato [ASCII PEM](https://www.ssl.com/guide/pem-der-crt-and-cer-x-509-encodings-and-conversions/) no [Real-Time Manager](https://manager.azion.com/) sem custo adicional. A chave privada não pode estar protegida por *passphrase*.
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/digital-certificates.mdx:38]:
```suggestion
Se você deseja usar um domínio próprio, você pode registrar seu próprio certificado TLS (X.509) e chave privada em formato [ASCII PEM](https://www.ssl.com/guide/pem-der-crt-and-cer-x-509-encodings-and-conversions/) no [Real-Time Manager](https://manager.azion.com/) sem custo adicional. A chave privada não pode estar protegida por *passphrase*.
```

File: src/content/docs/en/pages/main-menu/reference/build/edge-application/digital-certificates.mdx [hunk 7]
```diff
@@ -33,11 +33,11 @@ If you wish, you may use the assigned domain to deliver your static content over
...
 ---
 
-## Custom certificate
+## Edge certificate
 
-If you want to use a custom domain, you can register your own TLS certificate (X.509) at no additional cost in [Real-Time Manager](https://manager.azion.com). To use this feature, you'll need to prove that the domain is really yours.
+If you want to use a custom domain, you can add your own TLS certificate (X.509) and private key encoded in [ASCII PEM](https://www.ssl.com/guide/pem-der-crt-and-cer-x-509-encodings-and-conversions/) format to [Real-Time Manager](https://manager.azion.com) at no additional cost. The private key cannot be protected by passphrase.
 
-To use your Custom Certificate with Azion, the Server Name Indication (SNI) extension of the TLS protocol is used. Check the [browser list with SNI support](https://caniuse.com/#feat=sni).
+To use your custom certificate with Azion, the Server Name Indication (SNI) extension of the TLS protocol is used. Check the [browser list with SNI support](https://caniuse.com/#feat=sni).
```
Comment (bru-andrade) [inline src/content/docs/en/pages/main-menu/reference/build/edge-application/digital-certificates.mdx:40]:
```suggestion
When you bring your certificate to Azion, domains that use the certificate use the Server Name Indication (SNI) extension of the TLS protocol is used. Check the [browser list with SNI support](https://caniuse.com/#feat=sni).
```

File: src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/digital-certificates.mdx [hunk 8]
```diff
@@ -33,11 +33,11 @@ Se desejar, você pode utilizar o domínio atribuído para entregar seu conteúd
...
 ---
 
-## Custom certificate
+## Edge certificate
 
-Se você quiser alterar seu domínio, você pode registrar seu próprio certificado TLS (X.509) no [Real-Time Manager](https://manager.azion.com/) sem custo adicional. Para utilização desse recurso, você precisará comprovar a propriedade do domínio.
+Se você deseja usar um domínio próprio, você pode registrar seu próprio certificado TLS (X.509) e chave privada em formato [ASCII PEM](https://www.ssl.com/guide/pem-der-crt-and-cer-x-509-encodings-and-conversions/) no [Real-Time Manager](https://manager.azion.com/) sem custo adicional. A chave privada não pode estar protegida por *passphrase*.
 
-Para adicionar seu Custom Certificate, você precisará do par Certificado (X.509) e da respectiva chave privada, ambos no formato [ASCII PEM](https://www.google.com.br/search?q=Como+converter+um+certificado+PFX+para+PEM&cad=h). A chave privada não pode estar protegida por *passphrase*.
+Quando seu certificado é registrado com a Azion, edge applications com o certificado usam a extensão Server Name Indication (SNI) do protocolo TLS. Verifique a [lista da navegadores com suporte a SNI](https://caniuse.com/#feat=sni).
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/menu-principal/referencia/build/edge-application/digital-certificates.mdx:40]:
```suggestion
Quando seu certificado é registrado com a Azion, domínios com o certificado usam a extensão Server Name Indication (SNI) do protocolo TLS. Verifique a [lista da navegadores com suporte a SNI](https://caniuse.com/#feat=sni).
```

**Review by gabriel-azion — APPROVED** [review]

File: src/content/docs/en/pages/main-menu/reference/build/edge-application/digital-certificates.mdx [hunk 9]
```diff
@@ -33,11 +33,11 @@ If you wish, you may use the assigned domain to deliver your static content over
 
 ---
 
-## Custom certificate
+## Edge certificate
 
-If you want to use a custom domain, you can register your own TLS certificate (X.509) at no additional cost in [Real-Time Manager](https://manager.azion.com). To use this feature, you'll need to prove that the domain is really yours.
+If you want to use a custom domain, you can add your own TLS certificate (X.509) and private key encoded in [ASCII PEM](https://www.ssl.com/guide/pem-der-crt-and-cer-x-509-encodings-and-conversions/) format to [Real-Time Manager](https://manager.azion.com) at no additional cost. The private key cannot be protected by passphrase.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/main-menu/reference/build/edge-application/digital-certificates.mdx:38]:
```suggestion
If you want to use a custom domain, you can add your own TLS certificate (X.509) and private key encoded in [ASCII PEM](https://www.ssl.com/guide/pem-der-crt-and-cer-x-509-encodings-and-conversions/) format to [Real-Time Manager](https://manager.azion.com) at no additional cost. The private key cannot be protected by *passphrase*.
```

**Review by MarianaAguilera — APPROVED** [review]
just a suggestion

**Pull request merged**